### PR TITLE
Cert login save user info

### PR DESCRIPTION
**Pull request by Havkost**
## Description

Return first and last name as `userInfo` (was `user`) instead of name in login route

## Changes

- Return first and last name as `userInfo` (was `user`) instead of name in login route
- Removed legacy version of retrieving environment variables in keys.js.

## Related Issues

(#66 in educado-mobile)

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Any potential security implications have been considered and addressed.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

None.

## Notes for Reviewers

None.

### CODE REVIEW

**Review by Mafusn — APPROVED** [review]
Looks good, remember to update the tests

File: config/keys.js
Comment (MortenMunk) [inline config/keys.js:1]: I think this can be removed entirely?

**Review by MortenMunk — CHANGES_REQUESTED** [review]
I believe the config folder can be removed?

other than that, it looks good 👍 🥇

**Review by MaizeHH — APPROVED** [review]

**Review by MortenMunk — APPROVED** [review]
Looks good 👍 🥇

**Pull request merged**